gef.py

```diff
@@ -6157,6 +6166,28 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
         return
 
 
+@register
+class SoCommand(GenericCommand):
+    """Breaks on the instruction immediately following this one. Ex: Step over call instruction"""
+
+    _cmdline_ = "so"
```

Comment:
I agree with @Ordoviz `stepover` is more a meaningful name. And make `so` an alias

Reply:
done!